### PR TITLE
global: split of MARC21 demo records

### PR DESCRIPTION
**Pull request by jirikuncar**
- simplifies db management in tests
- enables SQLAlchemy modification tracking if not set by latest Flask-SQLAlchemy

/cc @drjova

### CODE REVIEW

File: invenio_records/ext.py [hunk 1]
```diff
@@ -68,6 +68,7 @@ def init_app(self, app,
 
     def init_config(self, app):
         """Initialize configuration."""
+        app.config.setdefault('SQLALCHEMY_TRACK_MODIFICATIONS', True)
```
Comment (lnielsen) [inline invenio_records/ext.py]: Shouldn't this go to invenio-db?

Comment (jirikuncar) [inline invenio_records/ext.py]: Invenio-DB doesn't really use this signal.

File: .travis.yml [hunk 2]
```diff
@@ -62,6 +62,8 @@ before_install:
   - "requirements-builder -e all --level=min setup.py > .travis-lowest-requirements.txt"
   - "requirements-builder -e all --level=pypi setup.py > .travis-release-requirements.txt"
   - "requirements-builder -e all --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "mysql -e 'CREATE DATABASE IF NOT EXISTS travis;' -uroot"
```
Comment (jirikuncar) [inline .travis.yml]: Ups, this was missing before.

File: invenio_records/ext.py (hunk 1 above)
Comment (jirikuncar) [inline invenio_records/ext.py]: Another reason is that if one does not use invenio-records then this config makes the performance worse globally when it's not necessary.

**Comment by jirikuncar** [comment]
@lnielsen it's ready :rocket:

**Pull request merged**